src/rb3/arkfile.hpp

```diff
@@ -3,11 +3,14 @@
 #include <types.h>
 class ArkFile {
 public:
+//	void fn_802E7500(s32 param_1, u32 param_2); // big headache. baaad.
+	u32 fn_802E76D8_stub();
```

Comment:
name this `ReturnsZero`

when we fix the linker we will properly define it and get it linked in